dash/dash.py

```diff
@@ -827,6 +827,41 @@ def _generate_meta_html(self):
 
         return "\n      ".join(tags)
 
+    def _pages_meta_tags(self):
+        start_page, path_variables = self._path_to_page(flask.request.path.strip("/"))
+        image = start_page.get("image", "")
+        if image:
+            image = self.get_asset_url(image)
+
+        title = start_page.get("title", self.title)
+        if callable(title):
+            title = title(**path_variables) if path_variables else title()
+
+        description = start_page.get("description", "")
+        if callable(description):
+            description = (
+                description(**path_variables) if path_variables else description()
+            )
+
+        return dedent(
+            f"""           
+            <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Would the regular index benefit from this meta too?

Reply:
absolutely!  I'll add it there instead.